app/helpers/mailer_helper.rb

```diff
@@ -127,4 +127,54 @@ def creator_text(work)
       work.pseuds.map { |p| text_pseud(p) }.to_sentence.html_safe
     end
   end
+
+  def work_metadata_label(text)
+    text.html_safe + t("mailer.general.metadata_label_indicator")
+  end
+
+  def work_tag_metadata_label(tags)
+    return if tags.empty?
+
+    type = tags.first.type
+    t("activerecord.models.#{type.underscore}", count: tags.count) + t("mailer.general.metadata_label_indicator")
+  end
+
+  # We don't use .to_sentence because these aren't links and we risk making any
+  # connector word (e.g., "and") look like part of the final tag.
+  def work_tag_metadata_list(tags)
```

Comment:
Is this used directly in a view? If so, we should make it private and keep the list of available helpers shorter.

Then we can have specs just for the public ones as well.